src/GitHub.Exports/Primitives/UriString.cs

```diff
@@ -197,7 +166,7 @@ public override UriString Combine(string addition)
             if (url != null)
             {
                 var urlBuilder = new UriBuilder(url);
-                if (!String.IsNullOrEmpty(urlBuilder.Query))
+                if (!string.IsNullOrEmpty(urlBuilder.Query))
```

Comment:
> Nope. I think it's my overeager R# that does this. We should pick an approach and stick to it. Do you have a preference?

Weeell, I always type String.Something out of habit, so I'm sure the code is way more sprinkled with the uppercase version than the lowercase one (and I doubt I can teach my fingers otherwise...) :smile:

> BTW, this has nothing to do with value type or class type. string is just the shorthand for System.String but it's the exact same class. :smile:

Yeah, I keep forgetting we're in .NET land where there are no crazy peeps doing weird stuff with jit and string/String differences...